ci_build/azure_pipelines/trimmed_keras2onnx_application_tests.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         INSTALL_KERAS: pip install keras==2.2.4
         UNINSTALL_KERAS:
         INSTALL_TENSORFLOW: pip install tensorflow==1.15.0
-        INSTALL_ORT: pip install onnxruntime==1.8.0
+        INSTALL_ORT: pip install -i https://test.pypi.org/simple/ ort-nightly
```

Comment:
Tensorflow is only compatible with `numpy==1.21.x` when it's greate than or equal to version `2.7.x`.